graylog2-web-interface/src/views/logic/widgets/Widget.ts

```diff
@@ -89,6 +97,7 @@ class Widget {
 
     return this.id === other.id
       && this.filter === other.filter
+      && isDeepEqual(this.filters, other.filters)
```

Comment:
Let's use the same checks here for `fitler` and `filters` like in `Query.ts`, just to be save:
```
|| !((isNullish(this.filter) && isNullish(other.filter)) || isDeepEqual(this.filter, other.filter))
|| !((isNullish(this.filters) && isNullish(other.filters)) || isDeepEqual(this.filters, other.filters))
```